parquet-column/src/test/java/org/apache/parquet/column/values/dictionary/TestDictionary.java

```diff
@@ -628,8 +628,8 @@ private void writeRepeated(int COUNT, ValuesWriter cw, String prefix) {
   }
 
   private void writeRepeatedWithReuse(int COUNT, ValuesWriter cw,
-                                      String prefix) throws UnsupportedEncodingException {
-    Binary reused = Binary.fromReusedByteArray((prefix + "0").getBytes("UTF-8"));
+                                      String prefix) {
```

Comment:
Now the function is much shorter, I would collapse it to a single line.

Reply:
Can do.  It's a test class however and I'm often told not to do re-formatting.